bedrock/base/templates/macros.html

```diff
@@ -474,27 +474,3 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
     </footer>
   </section>
 {%- endmacro %}
-
-
-{% macro google_custom_search(search_type, type_class, default_term, term_list) %}
```

Comment:
I seem to remember some warning from either @pmac or @jgmize about removing a shared macro sometimes causing issues with pages that are still cached. Not sure if this is still the case, but flagging here?